### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [llorephie/jekyll-bootstrap#1]
The user @carlsonsantana validate your site template "https://llorephie.me/jekyll-bootstrap/" and found these accessibility errors:
* Page https://llorephie.me/jekyll-bootstrap/:
  * Issue *3d6ebd78-d1f1-47bf-8715-f23c0d32a7a7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.63:1. Recommendation: change text colour to #967100.
    * **Selector**: `html > body > footer > div > div:nth-child(1) > p > a`
    * **Context**: `<a class="text-warning" href="/jekyll-bootstrap/feed.xml" rel="alternate">via RSS</a>`
  * Issue *d74b0c0f-b0aa-4159-b871-326876034141*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > footer > div > div:nth-child(1) > details > p:nth-child(3) > a`
    * **Context**: `<a href="https://creativecommons.org/licenses/by-sa/4.0/" rel="license">Content available under Creativ...</a>`
  * Issue *cbe67e2f-8a53-4869-b0d9-442f305499f5*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > footer > div > div:nth-child(1) > details > p:nth-child(2) > a`
    * **Context**: `<a href="mailto:mailbox@example.com">mailbox@example.com</a>`
  * Issue *96ab162e-2b17-4a7b-ac9d-557dba6a1bd5*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > div > div > main > section > article:nth-child(6) > a`
    * **Context**: `<a class="btn btn-outline-primary btn-lg btn-block" href="/jekyll-bootstrap/misc/2017/12/01/long.html">Read more</a>`
  * Issue *84b87942-2f06-406c-a229-59c9f91080e2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > div > div > main > section > article:nth-child(4) > a`
    * **Context**: `<a class="btn btn-outline-primary btn-lg btn-block" href="/jekyll-bootstrap/junk/2017/12/01/post-content.html">Read more</a>`
  * Issue *454d7a6d-6211-450e-8767-f19b83e63aa3*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > div > div > main > section > article:nth-child(2) > a`
    * **Context**: `<a class="btn btn-outline-primary btn-lg btn-block" href="/jekyll-bootstrap/2017/12/01/welcome-to-jekyll.html">Read more</a>`
  * Issue *66cb82fc-c078-4d7c-9cdf-2eee494724e5*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.82:1. Recommendation: change text colour to #3e3e3e.
    * **Selector**: `html > body > div > div > main > section > article:nth-child(2) > p > code:nth-child(2)`
    * **Context**: `<code class="highlighter-rouge">jekyll serve</code>`
  * Issue *43be36ab-748c-467d-9632-0691c40f2db7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.82:1. Recommendation: change text colour to #3e3e3e.
    * **Selector**: `html > body > div > div > main > section > article:nth-child(2) > p > code:nth-child(1)`
    * **Context**: `<code class="highlighter-rouge">_posts</code>`
  * Issue *e47eed0e-7f8d-4a51-9304-b7c201d46c2b*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.1.NoTitleEl
    * **Message**: A title should be provided for the document, using a non-empty title element in the head section.
    * **Selector**: `html > head`
    * **Context**: `<head>
    
...</head>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/62dd7841-63df-4a11-9944-b60025eb6da8/).
